usaspending_api/common/validator/helpers.py

```diff
@@ -102,14 +102,14 @@ def validate_datetime(rule):
     if "min" in rule:
         min_cap = datetime.datetime.strptime(rule["min"], dt_format)
         if value < min_cap:
-            logger.info("{}. Setting to {}".format(BELOW_MINIMUM_MSG.format(**rule), min_cap))
-            value = min_cap
+            message = rule.get("min_message", BELOW_MINIMUM_MSG).format(**rule)
+            raise UnprocessableEntityException(message)
 
     if "max" in rule:
         max_cap = datetime.datetime.strptime(rule["max"], dt_format)
         if value > max_cap:
-            logger.info(ABOVE_MAXIMUM_MSG.format(**rule))
-            value = max_cap
```

Comment:
Why are we using the sparser `BELOW_MINIMUM_MSG` when we have the more descriptive (and less redundant)`TIME_PERIOD_MIN_MESSAGE`?

Reply:
Because this function knows nothing about the type of date being submitted.  `TIME_PERIOD_MIN_MESSAGE` is very specific to advanced search dates.